src/schema/__tests__/partner_show.test.js

```diff
@@ -35,6 +35,73 @@ describe("PartnerShow type", () => {
     PartnerShow.__ResetDependency__("total")
   })
 
+  it("include false is_online_exclusive flag for shows with location", () => {
+    showData.location = "test location"
+    const query = `
+      {
+        partner_show(id: "new-museum-1-2015-triennial-surround-audience") {
+          is_online_exclusive
+        }
+      }
+    `
+    return runQuery(query).then(data => {
+      expect(data).toEqual({
+        partner_show: {
+          is_online_exclusive: false,
+        },
+      })
+    })
+  })
+  it("include false is_online_exclusive flag for shows with fair_location", () => {
+    showData.fair_location = "test location"
```

Comment:
In CMS I opted for checking for Fair in general as well as is_reference boolean + partner_city. I think in our current thinking we only talking about regular partner_shows that can be online only. Like I'm wondering if we somehow will end up with the show that has a fair but fair did not have a location that got stored into fair_location - can this show be considered online_exclusive?